picard/coverart/image.py

```diff
@@ -115,6 +134,13 @@ def data(self):
     def filename(self):
         return self._filename
 
+    def touch(self):
+        if self._filename:
+            try:
+                Path(self._filename).touch()
```

Comment:
I think using `touch()` alone isn't totally correct, because it may (re-)create the file if it was removed. Also, according to https://superuser.com/questions/187071/in-macos-how-often-is-tmp-deleted/187105#187105 the cleaner script checks for atimes too, but this is not reliable (dunno on macos, but on linux one can disable atime updates), so `touch()` is appropriate as it will change mtime.
So I think we just need to ensure the file still exists before touching it, there will be a race condition anyway since we use filename.

Also, I think we need something more generic that can be used for other temporary files or dirs, so instead of having a timer in cover art, and a code to touch the file I would create a "main" timer (running every X minutes), that touch a list of files build by a register function.

Something like:

```python
files_to_touch_on_regular_basis = set()

def  touch_on_regular_basis(filepath):
  files_to_touch_on_regular_basis.add(filepath)

def run_timer_touch_on_regular_basis():
   for filepath in files_to_touch_on_regular_basis:
       # check if filepath still exists
       # if it does, touch
       # if not, remove from files_to_touch_on_regular_basis and do nothing
```

Reply:
Ok, yes. That makes sense. I'll rework the patch and put that functionality into a separate module. This all feels a bit awkward still, but I think it is still the better solution than using a custom temp folder.

I'll also split the PR then and submit the first commit separately. I think both are really independent of each other, and I need a few days to find the time for reworking the temp file handling.